pkg/api/types.go

```diff
@@ -85,117 +74,12 @@ const (
-	GCLowThreshold                   int               `json:"gclowthreshold,omitempty"`
-	EtcdVersion                      string            `json:"etcdVersion,omitempty"`
-	EtcdDiskSizeGB                   string            `json:"etcdDiskSizeGB,omitempty"`
-	EtcdEncryptionKey                string            `json:"etcdEncryptionKey,omitempty"`
-	EnableDataEncryptionAtRest       *bool             `json:"enableDataEncryptionAtRest,omitempty"`
-	EnableEncryptionWithExternalKms  *bool             `json:"enableEncryptionWithExternalKms,omitempty"`
-	EnablePodSecurityPolicy          *bool             `json:"enablePodSecurityPolicy,omitempty"`
-	Addons                           []KubernetesAddon `json:"addons,omitempty"`
-	KubeletConfig                    map[string]string `json:"kubeletConfig,omitempty"`
-	ControllerManagerConfig          map[string]string `json:"controllerManagerConfig,omitempty"`
-	CloudControllerManagerConfig     map[string]string `json:"cloudControllerManagerConfig,omitempty"`
-	APIServerConfig                  map[string]string `json:"apiServerConfig,omitempty"`
-	SchedulerConfig                  map[string]string `json:"schedulerConfig,omitempty"`
-	CloudProviderBackoff             bool              `json:"cloudProviderBackoff,omitempty"`
-	CloudProviderBackoffRetries      int               `json:"cloudProviderBackoffRetries,omitempty"`
-	CloudProviderBackoffJitter       float64           `json:"cloudProviderBackoffJitter,omitempty"`
-	CloudProviderBackoffDuration     int               `json:"cloudProviderBackoffDuration,omitempty"`
-	CloudProviderBackoffExponent     float64           `json:"cloudProviderBackoffExponent,omitempty"`
-	CloudProviderRateLimit           bool              `json:"cloudProviderRateLimit,omitempty"`
-	CloudProviderRateLimitQPS        float64           `json:"cloudProviderRateLimitQPS,omitempty"`
-	CloudProviderRateLimitBucket     int               `json:"cloudProviderRateLimitBucket,omitempty"`
-	NonMasqueradeCidr                string            `json:"nonMasqueradeCidr,omitempty"`
-	NodeStatusUpdateFrequency        string            `json:"nodeStatusUpdateFrequency,omitempty"`
-	HardEvictionThreshold            string            `json:"hardEvictionThreshold,omitempty"`
-	CtrlMgrNodeMonitorGracePeriod    string            `json:"ctrlMgrNodeMonitorGracePeriod,omitempty"`
-	CtrlMgrPodEvictionTimeout        string            `json:"ctrlMgrPodEvictionTimeout,omitempty"`
-	CtrlMgrRouteReconciliationPeriod string            `json:"ctrlMgrRouteReconciliationPeriod,omitempty"`
-}
-
-// CustomFile has source as the full absolute source path to a file and dest
-// is the full absolute desired destination path to put the file on a master node
-type CustomFile struct {
-	Source string `json:"source,omitempty"`
-	Dest   string `json:"dest,omitempty"`
-}
-
 // OpenShiftConfig holds configuration for OpenShift
 type OpenShiftConfig struct {
```

Comment:
I don't think we use ConfigBundles anymore. Also the htpasswd options are not used, too, but probably we want a follow-up issue to use them? Same question for EnableAADAuthentication.

Reply:
nope, you're right, removing.  I went with the assumption that OpenShiftConfig was correct and didn't look at those.  Thanks!